RCSwitch.cpp

```diff
@@ -95,17 +94,14 @@ const unsigned int RCSwitch::nSeparationLimit = 4600;
 // according to discussion on issue #14 it might be more suitable to set the separation
 // limit to the same time as the 'low' part of the sync signal for the current protocol.
 unsigned int RCSwitch::timings[RCSWITCH_MAX_CHANGES];
-#endif
```

Comment:
The above changes, and similar changes below, are not acceptable. These `#ifdefs` are there for an good reason: To keep code size low for people who do not need to receive data.